src/wire/ipv6.rs

```diff
@@ -169,6 +169,18 @@ impl Address {
         }
         bytes
     }
+
+    /// The solicited node for the given unicast address.
+    ///
+    /// # Panics
+    /// This function panics if the given address is not
+    /// unicast.
```

Comment:
This function doesn't panic, as written.

Reply:
Good catch.

Reply:
This should panic as described in the documentation.